src/components/ha-analytics.ts

```diff
@@ -36,52 +45,101 @@ export class HaAnalytics extends LitElement {
           this.analytics.huuid
         )}
       </p>
-      ${PREFERENCES.map((preference) =>
-        preference === "base" || this.analytics.preferences.includes("base")
-          ? html`<ha-settings-row>
-              <span slot="prefix">
-                <ha-checkbox
-                  @change=${this._handleRowCheckboxClick}
-                  .checked=${this.analytics.preferences.includes(preference)}
-                  .preference=${preference}
-                >
-                </ha-checkbox>
-              </span>
-              <span slot="heading">
-                ${preference === "usage"
-                  ? isComponentLoaded(this.hass, "hassio")
-                    ? this.hass.localize(
-                        `ui.panel.config.core.section.core.analytics.preference.usage_supervisor.title`
-                      )
-                    : this.hass.localize(
-                        `ui.panel.config.core.section.core.analytics.preference.usage.title`
-                      )
+      <ha-settings-row>
+        <span slot="prefix">
+          <ha-checkbox
+            @change=${this._handleRowCheckboxClick}
+            .checked=${this.analytics.preferences.includes("base")}
+            .preference=${"base"}
+          >
+          </ha-checkbox>
+        </span>
+        <span slot="heading">
+          ${this.hass.localize(
+            `ui.panel.config.core.section.core.analytics.preference.base.title`
+          )}
+        </span>
+        <span slot="description">
+          ${this.hass.localize(
+            `ui.panel.config.core.section.core.analytics.preference.base.description`
+          )}
+        </span>
+      </ha-settings-row>
+      ${ADDITIONAL_PREFERENCES.map(
+        (preference) =>
+          html`<ha-settings-row>
+            <span slot="prefix">
+              <ha-checkbox
+                @change=${this._handleRowCheckboxClick}
+                .checked=${this.analytics.preferences.includes(preference)}
+                .preference=${preference}
+                .disabled=${!this.analytics.preferences.includes("base")}
+              >
+              </ha-checkbox>
+            </span>
+            <span slot="heading">
+              ${preference === "usage"
+                ? isComponentLoaded(this.hass, "hassio")
+                  ? this.hass.localize(
+                      `ui.panel.config.core.section.core.analytics.preference.usage_supervisor.title`
+                    )
                   : this.hass.localize(
-                      `ui.panel.config.core.section.core.analytics.preference.${preference}.title`
+                      `ui.panel.config.core.section.core.analytics.preference.usage.title`
+                    )
+                : this.hass.localize(
+                    `ui.panel.config.core.section.core.analytics.preference.${preference}.title`
+                  )}
+              ${!this.analytics.preferences.includes("base")
+                ? html`<ha-help-tooltip
```

Comment:
I would just make this a normal tooltip on the disabled element I think?

Reply:
This looks cooler :D